src/overlays/actors/ovl_En_Bombers/z_en_bombers.c

```diff
@@ -295,11 +295,11 @@ void func_80C03AF4(EnBombers* this, PlayState* play) {
             break;
     }
 
-    if (!Text_GetFaceReaction(play, 0x12)) {
+    if (!Text_GetFaceReaction(play, FACE_REACTION_SET_BOMBERS)) {
```

Comment:
```suggestion
    if (Text_GetFaceReaction(play, FACE_REACTION_SET_BOMBERS) != 0) {
```

Reply:
Did you mean `== 0`? Because doing `!= 0` here isn't equivalent (and doesn't match).

Reply:
Ah yeah :), that should be a `==0`